src/components/NewsWidget/newsWidget.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { NewsWidget } from '.';
+
+export default {
+  component: NewsWidget,
+  title: 'Pangolin/NewsWidget',
+};
+
+const TemplateNewsWidget: ComponentStory<typeof NewsWidget> = () => <NewsWidget />;
```

Comment:
Storybook is not working

<img width="1440" alt="Screenshot 2022-05-18 at 5 39 29 PM" src="https://user-images.githubusercontent.com/6604146/169035482-764639ea-c11b-40f4-aa48-0fa7c55ee691.png">

Reply:
I deleted this file as the pb you see is the same I have with other stories that you do not seem to have on you machine...